main.rb

```diff
@@ -0,0 +1,6 @@
+require './app'
```

Comment:
- [ ] I got an error when I tried to run the app. Please see screenshot of the error

Screenshot | 
:--------------:|
![a1](https://user-images.githubusercontent.com/39523312/166105610-d9021aed-a35a-40f6-8fb9-cba011da5316.png)